src/Components/Votes/VoteRecord/index.js

```diff
@@ -32,6 +34,21 @@ export class VoteRecordComponent extends React.Component {
             <div>
               Result: {voteRecord.althingi_result}
             </div>
+            <div
+              style={{
+                display: 'flex',
+                flexWrap: 'wrap',
+                justifyContent: 'space-between',
+              }}
+            >
+              {voteRecord.votes.map(vote =>
+                <PoliticianCard
+                  key={vote.parliament_member.id}
+                  style={{ backgroundColor: VOTE_STATUS_TO_COLOR[vote.althingi_result] }}
```

Comment:
Does it not make more sense to pass the `vote.althingi_result` as a prop to `PoliticianCard`
Which then calls `VOTE_STATUS_TO_COLOR` ? (this constant could be moved to a global `theme.js` or `constants.js` file

Reply:
We wont have a `althingi_result` everywhere where we use the `PoliticianCard` (like when we just want to list all Politcians). 

Not sure what the best thing to do here is - could have a styling function that takes in the result and returns the correct styling but that feels a bit weird

Reply:
Otherwise it might be worth trying `CSSStyleSheet` from https://www.saltycrane.com/flow-type-cheat-sheet/latest/

Reply:
that will work - my only concern right now is that it might explode if we add a lot more styles as props - but we can evaluate it at that point